docs/faq/README.md

```diff
@@ -69,17 +69,16 @@ spec:
     servicePort: 80
 ```
 
-Setting the annotation to any value other than "gce" or the empty string, will
-force the GCE controller to ignore your Ingress. The same applies for the nginx
-controller.
+The GCE controller will only act on Ingresses with the annotation value of "gce" or empty string "" (the default value if the annotation is omitted).
 
-To completely stop the Ingress controller on GCE/GKE, please see [this](gce.md#host-do-i-disable-the-ingress-controller) faq.
+The nginx controller will only act on Ingresses with the annotation value of "nginx".
```

Comment:
Please confirm that this is the behavior.

Reply:
@tonglil nginx controller will only act on Ingresses with the annotation value of "nginx" and empty class annotation

Reply:
Ok cool. Just to confirm, both GCE and nginx controllers will act on empty string annotation?

Reply:
exactly

Reply:
Updated.